mgmt/rest/v2/mock/mock_config_manager.go

```diff
@@ -70,19 +70,25 @@ func (MockConfigManager) DeletePluginConfigDataNodeFieldAll(fields ...string) cd
 // rest_v2_test.go on the plugin config routes found in mgmt/rest/server.go
 const (
 	SET_PLUGIN_CONFIG_ITEM = `{
-  "user": "Jane"
+  "config": {
```

Comment:
Can we avoid the breaking API change?  We should avoid changing response objects.